src/components/Todos/Todos.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import Todo from './Todo/Todo';
+
+const todos = (props) => {
+  return (
+    <section className="main">
+      <input
+        className="toggle-all"
+        type="checkbox" />
+      <label htmlFor="toggle-all" ></label>
+      <ul className="todo-list">
+        {props.todos.map((item) =>(
+          <Todo
+            onDeleteTodo={props.onDeleteTodo}
+            completedHandler={props.completedHandler}
+            key={item.id}
```

Comment:
why do you need `key` in loop UI?

Reply:
- Map method
- Keys help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity. Each children have a unique key is very helpful when we pick an element in our list.
- According to ReactJS.org, when you render a list several times ReactJS can match the elements and mutate every child instead of realizing it can keep the children subtrees intact. If we not have a unique key, any change of the order can affect the performance .